test/test_permeation_problem.py

```diff
@@ -76,13 +70,12 @@ def siverts_law(T, S_0, E_S, pressure):
     )
     bc_outgas = dirichletbc(Constant(my_mesh.mesh, PETSc.ScalarType(0)), right_dofs, V)
     my_model.boundary_conditions = [bc_sieverts, bc_outgas]
-
-    final_time = 50
-
-    # log.set_log_level(log.LogLevel.INFO)
+    my_model.create_solver()
```

Comment:
if this is what is needed for the test to work, then I'd like the call to `.create_solver()` to be removed from `.initialise()` for now.